packages/blocks/src/api/parser/convert-legacy-block.js

```diff
@@ -67,5 +72,8 @@ export function convertLegacyBlockNameAndAttributes( name, attributes ) {
 		name = 'core/comment-date';
 	}
 
-	return [ name, newAttributes ];
+	return applyFilters( 'editor.convertLegacyBlockNameAndAttributes', [
```

Comment:
```suggestion
	return applyFilters( 'blocks.convertLegacyBlockNameAndAttributes', [
```

The other filters in the `@wordpress/block` package use the `blocks` prefix.